src/shared/components/VideoPreviewBase/VideoPreviewBase.tsx

```diff
@@ -202,12 +261,33 @@ const VideoPreviewBase: React.FC<VideoPreviewBaseProps> = ({
                     <SpacedPlaceholder height={main ? 16 : 12} width={main ? '40%' : '80%'} />
                   ) : createdAt ? (
                     <MetaText variant="subtitle2" main={main} scalingFactor={scalingFactor}>
-                      {formatVideoViewsAndDate(views ?? null, createdAt, { fullViews: main })}
+                      {/* TODO: draft one should not be createdAt ?? */}
+                      {videoPublishState === 'draft'
+                        ? `Last updated ${formatDateAgo(createdAt)}`
```

Comment:
@kdembler I think here we need something else from the query node that tells us when the video was last updated, not sure if it already exists

Reply:
Nvm I think this all will be saved in localStorage

Reply:
Yeah, `last updated` is displayed only for drafts and we will get that from local storage